phpBB/includes/acp/acp_board.php

```diff
@@ -196,6 +196,7 @@ function main($id, $mode)
 						'allow_post_flash'		=> array('lang' => 'ALLOW_POST_FLASH',		'validate' => 'bool',	'type' => 'radio:yes_no', 'explain' => true),
 						'allow_smilies'			=> array('lang' => 'ALLOW_SMILIES',			'validate' => 'bool',	'type' => 'radio:yes_no', 'explain' => false),
 						'allow_post_links'		=> array('lang' => 'ALLOW_POST_LINKS',		'validate' => 'bool',	'type' => 'radio:yes_no', 'explain' => true),
+						'allowed_schemes_links'	=> array('lang' => 'ALLOWED_SCHEMES_LINKS',	'validate' => 'string',	'type' => 'text:0:255', 'explain' => true),
```

Comment:
No option in 3.1, any scheme is allowed except for `javascript` and I think any scheme that ends in "script" too.

If the config option is not displayed in the ACP's UI, it might as well be hardcoded. An extension would still be able to configure which schemes are allowed via the `core.text_formatter_s9e_configure_after` event.